README.md

```diff
@@ -150,6 +150,24 @@ The above steps will deploy the statically generated API docs by running the `np
     - dev environment: Singapore (`ap-southeast-1`)
     - prod environment: Mumbai (`ap-south-1`)
 
+## Github Actions
+### Steps for CI/CD :
```

Comment:
```suggestion
### Steps to CI/CD using GitHub Actions:

The following is the list of steps that is executed as a part of the CICD setup
```

Reply:
Addressed this suggestion in latest commit